caculate percent commission/main.js

```diff
@@ -0,0 +1,13 @@
+calPerCommission = (sales) => {
+    var sales = prompt('Nhập doanh số bán được: ')
+    if (sales <= 100000000) {
+        alert('Số hoa hồng bạn nhận được là: ' + sales*0.05)
```

Comment:
```suggestion
        alert('Số hoa hồng bạn nhận được là: ' + sales * 0.05)
```
nhìn kỹ cách đjăt dấu cách